### PR TITLE
chore: lint code and update dependencies

### PR DESCRIPTION
**Pull request by rsmayda**
Issue #, if available:

Description of changes:
- update lint dependencies
- lint code
- update FWoA dependencies

Checklist:

* [x] Have you successfully deployed to an AWS account with your changes?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by ssvegaraju — APPROVED** [review]
Looks good to me. Is there a specific reason to pin serverless version to 2.29?

File: auditLogMover/package.json [hunk 1]
```diff
@@ -5,22 +5,22 @@
...
-    "eslint-plugin-import": "^2.22.1",
-    "eslint-plugin-prettier": "^3.1.2",
+    "eslint": "^7.32.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-plugin-import": "^2.24.2",
+    "eslint-plugin-prettier": "^4.0.0",
     "jest": "^26.6.3",
-    "serverless": "^1.73.1",
+    "serverless": "2.29.0",
```
Comment (carvantes) [inline auditLogMover/package.json:23]: Did you test sls offline?

Comment (rsmayda) [reply auditLogMover/package.json:23]: This is for auditLogMover and I don't believe supports sls-offline

I did deploy it successfully

**Comment by rsmayda** [comment]
> Looks good to me. Is there a specific reason to pin serverless version to 2.29?

It is what is used in the main package.json. I wanted to have a consistent version across our apps

File: auditLogMover/package.json (hunk 1 above)
Comment (carvantes) [reply auditLogMover/package.json:23]: I see. I didn’t realize it was auditlogmover

**Pull request merged**